### ISSUE TITLE
Accent issue when typing on a list

### ISSUE THREAD

**Issue by ornanovitch** [askvortsov1/flarum-rich-text#14]
Hi Alexander, I hope you're doing well!

I think I already posted this problem somewhere on Discuss but it's worth opening an issue. So here is the problem I have on my forum on linux/firefox and this keyboard layout, which is pretty standard for FR keyboards:

```bash
❯ setxkbmap -query
rules:      evdev
model:      pc105
layout:     fr
variant:    latin9
```

Whenever I'm on a list, typing an accented character with a key combination (e.g. `^`, `¨`) put the accent before the character (e.g. `^e` in place of `ê`). Outside lists, everything's working. Note that I can't reproduce on the Prosemirror demo.

See the attached gif.

https://s9.gifyu.com/images/essai.gif

**Comment by askvortsov1** [comment]
I'm having trouble replicating your setup, and I'm not sure exactly which keys to press, since I'm not familiar with the french keyboard layout. Beyond `setxkbmap fr`, what setup do I need to do, and which keys should I use?

**Comment by ornanovitch** [comment]
I guess you should be able to reproduce this with an (virtual) azerty keyboard on linux, with the default variant.

With such a setup, normally you can for example produce a `â` by pressing on the `^` key and *then* on the `A` key, knowing that `^` is right to `P`, and that `A` is on `Q` on a qwerty.

The real strange thing is that it works normally with the Prosemirror demo. Do you need anything else? I guess I should try on different setups...

**Comment by askvortsov1** [comment]
Fascinating... I just tried it locally on my Flarum Rich Text instance, and that case (^a) worked perfectly: the ^ didn't show up, and when I pressed a, it was formatted properly into `â`. Are you using the latest version of rich text?

**Comment by ornanovitch** [comment]
I think but I'm going to check again. Are you sure you are trying to reproduce onto a list?

For me

```
ââââ
```

is working but

```
- ââââ
```

isn't (instead it produces ^a^a^a and renders an exponent)

**Comment by ornanovitch** [comment]
I just updraded with no luck unfortunately :/

**Comment by askvortsov1** [comment]
Yep, I checked on a list. Checked again on Firefox, if I press ^ once before a it works, it I press ^ twice it does what you described.

Are you able to replicate on various devices?

**Comment by ornanovitch** [comment]
Ok so this is really odd..

I just checked with a friend who's on Gentoo (I'm on Arch) with Firefox, he has the very same issue. We have different fr-keyboard variant so it doesn't seems to be a keyboard layout issue. I just tried with Chromium: same issue, so it's not a browser problem. I checked with Windows 10 and Firefox within a VM: no problem!

I don't really figure out how to debug this one...
